linalg/dtensor.hpp

```diff
@@ -43,9 +43,7 @@ class TensorInd
 public:
    MFEM_HOST_DEVICE static int result(const int* sizes, T first, Args... args)
    {
-#ifndef MFEM_USE_CUDA
```

Comment:
I think we do need these if**n**def-s to avoid calling the `MFEM_ASSERT` host function.

Reply:
Oops, added them back!